x-pack/plugins/spaces/server/usage_collection/spaces_usage_collector.ts

```diff
@@ -168,6 +173,11 @@ export function getSpacesUsageCollector(
     schema: {
       usesFeatureControls: { type: 'boolean' },
       disabledFeatures: {
+        stackAlerts: { type: 'long' },
```

Comment:
I wonder if we might want to consider using `DYNAMIC_KEY` here, so we don't need to maintain the list. On the other side, we know each feature that we add. What do you think?

Reply:
I think `DYNAMIC_KEY` makes sense here. Features are changing more frequently than I'd like, and having to maintain this list is error prone

Reply:
yeah! I agree! I guess the question is: do we want this to fail CI when any plugin uses the `features` API, so they need to add themselves to this list in the same PR.

**Pros:** We are explicit about the existing features we ship on each release. Which helps downstream (or does it, @elastic/infra-telemetry?).
**Cons:** Noise! Updating this list will ping the Security and Telemetry teams in the PR, adding [_unnecessary?_] overhead to the review process.

Reply:
FYI: The more I think about it, the more it makes sense to use `DYNAMIC_KEY`. I've tentatively added it. I'm happy to revert this change if we think otherwise.

Reply:
👍 no objections here